iceoryx_utils/include/iceoryx_utils/internal/objectpool/objectpool.hpp

```diff
@@ -384,7 +384,7 @@ class ObjectPool
 
     Index_t pointerToIndex(T* ptr)
     {
-        // todo: not critical, but refactor to use intptr/uintptr_t for pointer arithmetic later
+        /// todo: not critical, but refactor for pointer arithmetic later
```

Comment:
@todo here, at least a github issue should be created to track this

Reply:
There are a lot of todos and notes in there. Additionally, we do not use the objectpool anywhere so I created issue #66 to follow up on that. Either we clean it up when it is used somewhere or we remove it completely.

Reply:
thanks for creating!